.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+# This is a workflow to verify that Android build is successful.
```

Comment:
Same as above. I guess, as to keep the separately is going to break the commit history.